app/views/layouts/_header.html.erb

```diff
@@ -63,6 +63,7 @@
                     <li><%= link_to t('.edit_profile'), edit_user_path %></li>
                     <% if current_user.admin? %>
                       <li><%= link_to t('.admin'), '#admin_path', target: '_blank' %></li>
+                      <li><%= link_to t('.edit_frontpage'), edit_frontpage_path %></li>
```

Comment:
This has some weird consequences with the styling ![](https://i.gyazo.com/fca0dfc8ff9d72f10845ed58b9e0732e.png)